pkgs/by-name/st/streamlink/package.nix

```
@@ -39,19 +42,29 @@ python3Packages.buildPythonApplication rec {
     "test_no_cache"
   ];
 
-  propagatedBuildInputs = with python3Packages; [
-    certifi
-    isodate
-    lxml
-    pycountry
-    pycryptodome
-    pysocks
-    requests
-    trio
-    trio-websocket
-    urllib3
-    websocket-client
-  ];
+  propagatedBuildInputs =
+    with python3Packages;
+    [
+      certifi
+      isodate
+      lxml
+      pycountry
+      pycryptodome
+      pysocks
+      requests
+      trio
+      trio-websocket
+      urllib3
+      websocket-client
+    ]
+    ++ lib.attrVals extras optional-dependencies;
+
+  optional-dependencies = with python3Packages; {
+    decompress = [
+      urllib3.optional-dependencies.brotli
+      urllib3.optional-dependencies.zstd
+    ];
```

```suggestion
    decompress = 
      urllib3.optional-dependencies.brotli
      ++ urllib3.optional-dependencies.zstd;
```

or maybe use `lib.concatLists`